.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -123,6 +123,7 @@ From "Keep a Changelog":
 - [ ] Set the tag name to the version tag, for example: `1.0.0-alpha.0`
 - [ ] Set the release to target the `main` branch
 - [ ] Mark the release as 'pre-release' (until we are no longer alpha/beta)
+- [ ] Create a new PR to update the CHANGELOG.md file with the final change log.
```

Comment:
Can we make sure that the changelog is included in the tagged release?

If we create a new PR as the last step, it probably won't be merged before we tag the release. If could do it after "Update the draft release with the final changelog", and wait for the PR, then the updated changelog will be in the release tag.

Reply:
So the other option I was evaluating was to add the changelog update to the version update PR to get them both approved and merged at the same time but I didn't like that because they are really 2 separate things. 

If we do it after "Update the draft release with the final changelog" then we run the risk of the draft release in github being overwritten by any new merges to the repo. 

I guess the best thing to do would be to merge the separate Changelog PR _before_ "Update the draft release with the final changelog" and wait for that to be approved and merged before tagging the final release. That way we also will no longer need the riseup pad for the draft changelog.

I will update the instructions for a further review.